### ISSUE TITLE
When there are no checks deployment isn't allowed

### ISSUE THREAD

**Issue by redoz** [github/branch-deploy#191]
### Describe the Issue

A changeset that doesn't trigger any checks cannot be deployed;
![image](https://github.com/github/branch-deploy/assets/1670030/ecede9e7-6afe-4eab-a956-ac4ab7a93399)

We've changed one of our bicep files (and of course you could argue we should have a compile check on those, but we don't currently) and as such it doesn't let us deploy.

### Action Configuration

```yaml
      - name: Branch deployment
        id: branch-deploy
        uses: github/branch-deploy@v7.3.0
        with:
          # see https://github.com/github/branch-deploy#inputs-
          environment: qa # the default environment
          environment_targets: "qa"
          skip_reviews: "qa"
          environment_urls: "qa|https://qa.example.com"
          update_branch: disabled
          merge_deploy_mode: false
          stable_branch: master
          allow_forks: false
          environment_url_in_comment: true


```

**Comment by GrantBirki** [comment]
@redoz I know why this is happening. Its a bug and you stumbled across it. Thank you for reporting! I will work on getting a new release out with the fix right away :bow:

**Comment by GrantBirki** [comment]
@redoz Fixed in release [`v7.3.1`](https://github.com/github/branch-deploy/releases/tag/v7.3.1) :tada:

**Comment by redoz** [comment]
> @redoz Fixed in release [`v7.3.1`](https://github.com/github/branch-deploy/releases/tag/v7.3.1) :tada: 

Will update tomorrow, thanks for the quick fix!

**Comment by redoz** [comment]
@GrantBirki Can verify that it now allows for deployments on PRs that have no checks. Awesome, thank you!
